test/intl402/NumberFormat/prototype/resolvedOptions/return-keys-order-default.js

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.numberformat.prototype.resolvedoptions
+description: order of property keys for the object returned by resolvedOptions()
+features: [Intl.NumberFormat-v3]
+---*/
+
+assert.sameValue(
```

Comment:
This also should use `assert.compareArray`, and also should cover _all_ properties.